pkgs/shells/bash/5.nix

```diff
@@ -278,7 +278,7 @@ lib.warnIf (withDocs != null)
       mainProgram = "bash";
       identifiers.cpeParts =
         let
-          versionSplit = lib.split "p" version;
+          versionSplit = lib.split "p" fa.version;
```

Comment:
This PR *does* require that `meta` successfully evaluate. I'm not sure why, but `version` here somehow has a strange value like "20250101-dirty". Pinning the version to the final fixed point value resolves the issue.